src/components/Launch/LaunchForm/test/LaunchTaskForm.test.tsx

```diff
@@ -398,234 +425,73 @@ describe('LaunchForm: Task', () => {
         });
 
         describe('Auth Role', () => {
-            it('should require a value', async () => {
-                const { container, getByLabelText } = renderForm();
-                const roleInput = await waitFor(() =>
-                    getByLabelText(roleTypes.iamRole.inputLabel, {
-                        exact: false
-                    })
-                );
-
-                fireEvent.click(getSubmitButton(container));
-                await waitFor(() => expect(roleInput).toBeInvalid());
+            describe('should show correct label/helper text', () => {
+                it('for IAM', async () => {
+                    const { getByText, getByLabelText } = renderForm();
+                    await waitFor(() =>
+                        getByLabelText(
+                            AuthRoleStrings[AuthRoleTypes.IAM].inputLabel,
+                            {
+                                exact: false
+                            }
+                        )
+                    );
```

Comment:
What's this for?
Are we missing `expect` or is this for some other purpose?

Reply:
If you look the expect() is on the next line.  We need the async bit because these elements are rendered dynamically so we wait for the element to exist then we query it.